### ISSUE TITLE
Automatic update errors on launch

### ISSUE THREAD

**Issue by szTheory** [szTheory/exifcleaner#1]
```
Unhandled Promise Rejection

Error: Redirect was cancelled
    at SimpleURLLoaderWrapper.<anonymous> (electron/js2c/browser_init.js:2558:23)
```

It's probably a configuration issue with `electron-build` for automatic updates.

**Comment by szTheory** [comment]
Error appears to be caused by Electron, will be fixed in 7.2 (currently 7.1.x) https://github.com/sindresorhus/got/issues/899

**Comment by szTheory** [comment]
Related issue https://github.com/electron-userland/electron-builder/issues/4505

**Comment by szTheory** [comment]
Released update 2.0.0. Rolling back to Electron 7.1.2 may have fixed it no guarantees. Will find out if auto-update works in the next point release.

**Comment by szTheory** [comment]
Supposedly Electron 7.1.8 fixed this. Will re-open the issue if it pops up again.
